core/src/test/resources/json-ld.org/toRdf-manifest.jsonld

```diff
@@ -807,6 +807,83 @@
       "purpose": "Proper (re-)labeling of blank nodes if used with reverse properties.",
       "input": "toRdf-0119-in.jsonld",
       "expect": "toRdf-0119-out.nq"
+    }, {
+      "@id": "#t0120",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ToRDFTest"],
+      "name": "IRI Resolution (0)",
+      "purpose": "IRI resolution according to RFC3986.",
+      "input": "toRdf-0120-in.jsonld",
+      "expect": "toRdf-0120-out.nq"
+    }, {
+      "@id": "#t0121",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ToRDFTest"],
+      "name": "IRI Resolution (1)",
+      "purpose": "IRI resolution according to RFC3986.",
+      "input": "toRdf-0121-in.jsonld",
+      "expect": "toRdf-0121-out.nq"
+    }, {
+      "@id": "#t0122",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ToRDFTest"],
+      "name": "IRI Resolution (2)",
+      "purpose": "IRI resolution according to RFC3986.",
+      "input": "toRdf-0122-in.jsonld",
+      "expect": "toRdf-0122-out.nq"
+    }, {
+      "@id": "#t0123",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ToRDFTest"],
+      "name": "IRI Resolution (3)",
+      "purpose": "IRI resolution according to RFC3986.",
+      "input": "toRdf-0123-in.jsonld",
+      "expect": "toRdf-0123-out.nq"
+    }, {
+      "@id": "#t0124",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ToRDFTest"],
+      "name": "IRI Resolution (4)",
+      "purpose": "IRI resolution according to RFC3986.",
+      "input": "toRdf-0124-in.jsonld",
+      "expect": "toRdf-0124-out.nq"
+    }, {
+      "@id": "#t0125",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ToRDFTest"],
+      "name": "IRI Resolution (5)",
+      "purpose": "IRI resolution according to RFC3986.",
+      "input": "toRdf-0125-in.jsonld",
+      "expect": "toRdf-0125-out.nq"
+    }, {
+      "@id": "#t0127",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ToRDFTest"],
+      "name": "IRI Resolution (7)",
+      "purpose": "IRI resolution according to RFC3986.",
+      "input": "toRdf-0127-in.jsonld",
+      "expect": "toRdf-0127-out.nq"
+    }, {
+      "@id": "#t0129",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ToRDFTest"],
+      "name": "IRI Resolution (9)",
+      "purpose": "IRI resolution according to RFC3986.",
+      "input": "toRdf-0129-in.jsonld",
+      "expect": "toRdf-0129-out.nq"
+    }, {
+      "@id": "#t0130",
```

Comment:
Tests 0130, 0131 an 0132 do not appear in the official test suite at https://github.com/json-ld/json-ld.org/blob/master/test-suite/tests/toRdf-manifest.jsonld

Is there a plan to contribute them back there?